Makefile

```diff
@@ -9,4 +9,4 @@ format:
 	black . && isort .
 
 lint:
-	black --check . && isort --check . && flake8 .
+	black --check . && isort --check . && flake8 . && mypy .
```

Comment:
Good, but I would suggest changing it to something like this:
```
echo "Checking code format with black..."
black --check .
echo "Checking imports order..."
isort --check
echo "Running flake8..."
flake8 .
echo "Running mypy..."
```

Reply:
Wouldn't it be better to split those actions into different Makefile targets then?

Reply:
Sure